README.md

```diff
@@ -188,6 +189,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Softhover](https://www.softhover.com/) | Bansbari Rd, Kathmandu | Mobile Apps and Web Development Services
 - [Sofwena Pvt. Ltd.](https://sofwena.com/) | Patan Dhoka, Lalitpur | Sofwena is a company providing software development and product delivery services to Business House Holds, Agriculture Industry and Financial Management
 - [Source Code Pvt. Ltd.](www.sourcecode.com.np/) | Durbar Marg, Kathmandu | Financial technology company established with the aim of simplifying technology for capital markets
+- [SoAni Tech Pvt. Ltd.](www.soanitech.com) | Manbhawan, Lalitpur | Software Development Company 
```

Comment:
@milanshresta Please provide a proper description of the company. The description is very generic here.